torch/csrc/lazy/ts_backend/ts_backend_impl.cpp

```diff
@@ -99,6 +105,14 @@ class TSBackendImpl : public torch::lazy::BackendImplInterface {
     return std::make_shared<TSData>(scalar, device);
   }
 
+  torch::lazy::BackendDataPtr GetComputationDataFromNode(Node* node) const {
```

Comment:
@JackCaoG have you seen this new backend API proposed? i think it is a good idea. will you be OK implementing it?

Reply:
yea, I think it is cleaner than trying to dynamiccast in the ltc core, pt/xla can implement it.